simtools/camera_efficiency.py

```diff
@@ -262,41 +266,54 @@ def analyze(self, export=True, force=False):
         self._has_results = True
 
         print("\33[40;37;1m")
-        self._logger.info(f"Spectrum weighted reflectivity: {self.calc_reflectivity():.4f}")
-        self._logger.info(
-            f"Camera nominal efficiency with gaps (B-TEL-1170): {self.calc_camera_efficiency():.4f}"
-        )
-        self._logger.info(
-            "Telescope total efficiency"
-            f" with gaps (was A-PERF-2020): {self.calc_tel_efficiency():.4f}"
-        )
-        self._logger.info(
-            (
-                "Telescope total Cherenkov light efficiency / sqrt(total NSB efficency) "
-                "(A-PERF-2025/B-TEL-0090): "
-                f"{self.calc_tot_efficiency(self.calc_tel_efficiency()):.4f}"
-            )
-        )
-        self._logger.info(
-            "Expected NSB pixel rate for the reference NSB: "
-            f"{self.calc_nsb_rate():.4f} [p.e./ns]"
-        )
+        self._logger.info(f"\n{self.results_summary()}")
         print("\033[0m")
 
         if export:
             self.export_results()
 
     # END of analyze
 
+    def results_summary(self):
+        """
+        Print a summary of the results.
+        Include a header for the zenith/azimuth settings and the NSB spectrum file which was used.
+        The summary includes the various CTAO requirements and the final expected NSB pixel rate.
+        """
+        summary = (
+            f"Results summary for {self._telescope_model.name} at "
+            f"zenith={self.config.zenith_angle:.1f}, "
+            f"azimuth={self.config.azimuth_angle:.1f}\n"
+            f"Using the NSB spectrum file: {self.config.nsb_spectrum}\n"
+            f"\nSpectrum weighted reflectivity: {self.calc_reflectivity():.4f}\n"
+            "Camera nominal efficiency with gaps (B-TEL-1170): "
+            f"{self.calc_camera_efficiency():.4f}\n"
+            "Telescope total efficiency"
+            f" with gaps (was A-PERF-2020): {self.calc_tel_efficiency():.4f}\n"
+            "Telescope total Cherenkov light efficiency / sqrt(total NSB efficency) "
+            "(A-PERF-2025/B-TEL-0090): "
+            f"{self.calc_tot_efficiency(self.calc_tel_efficiency()):.4f}\n"
+            "Expected NSB pixel rate for the reference NSB: "
+            f"{self.calc_nsb_rate():.4f} [p.e./ns]\n"
+        )
+
+        return summary
+
     def export_results(self):
         """Export results to a csv file."""
```

Comment:
We should change here as well. First, it should be ecsv and not csv, and we should include also the summary

Reply:
I fixed the csv --> ecsv.
I don't want to combine the summary with this table since this table is not really necessary once the summary is available. We save the table as some sort of provenance, but what is actually important is the summary.